wordpress-mobile/WordPress-iOS
# [Stats Refresh] Post Stats: add 'View more' support

Ref #11189

This adds support for `View more` on the Post Stats `Months and Years` and `Avg Views Per Day` cards.
- Selecting `View more` will show all available years.
- Just like on the main cards, expanding a year shows the data rows for that year.

To test:
- Tip: en.blog > Home Page and the Field Guide > Home Page have data for several years.
- Access Post Stats via:
  - Insights > Latest Post Summary > View more (although this post probably won't have more than 6 years of views)
  - Period > Posts and Pages > row selection
  - Period > Posts and Pages > View more > row selection
- Select `View more` on `Months and Years` and `Avg Views Per Day`.
- Verify all years display.

![post_stats_view_more](https://user-images.githubusercontent.com/1816888/56999788-ef2fb800-6b6d-11e9-86ab-44e38732ae6c.png)
